### PR TITLE
add examples of getting response header information

### PR DESCRIPTION
**Pull request by christiancompton**
This PR adds examples of how to get header information from the operation responses in Python and Go.  This was already covered in the Node and Java examples.

### CODE REVIEW

**Comment by christiancompton** [comment]
As requested in https://github.ibm.com/arf/planning-sdk-squad/issues/1543

File: README.md [hunk 1]
```diff
@@ -722,6 +722,15 @@ Each operation will return the following values:
 3. `err` - An error object.  This will be nil if the operation was successful, or non-nil  
 if an error occurred.
 
+Here is an example of how to access the response and get additional Information
+beyond the response object:
+
+```go
+result, detailedResponse, err := myService.GetResource(options)
+
+responseHeaders := detailedResponse.GetHeaders()
```
Comment (padamstx) [inline README.md:731]: add an example of retrieving a specific header from `responseHeaders` (use same header name in each language's example).

File: README.md [hunk 2]
```diff
@@ -781,6 +790,8 @@ beyond the response object:
 ```python
 detailedResponse = my_service.get_resource(resource_id='resource-id-1')
 print(detailedResponse)
+
+responseHeaders = detailedResponse.get_headers()
```
Comment (padamstx) [inline README.md:794]: Add example showing how to retrieve a specific header from `responseHeaders`.

**Comment by christiancompton** [comment]
@padamstx I have updated the PR to get the `response-id` header in each language example.

**Comment by padamstx** [comment]
LGTM

**Pull request merged**